modules/ROOT/Edits/endpoint-https-send.adoc

```diff
@@ -0,0 +1,98 @@
+= HTTP and HTTPS Send Endpoint Settings
+
+HTTP and HTTPS send endpoint settings configure where to send messages in the following contexts:
+
+* Send to partners
++
+Target endpoint in an outbound message flow to which translated B2B/EDI messages are sent to the partners either directly or via a 3rd party connection
++
+* Target at host
++
+Target endpoint in an inbound message flow to which translated application JSON and XML messages are sent to the backend systems
+
+A host, partner, or 3rd party connection can own HTTP and HTTPS send endpoints.
+
+Create HTTP and HTTPS send endpoints by configuring these settings on the host, partner, or 3rd party connection page:
+
+[%header%autowidth.spread]
+|===
+|Name |Description |Required
+| Owner
+| Partner, 3rd party connection, or host who owns the endpoint
+| Yes
+
+|Protocol
+| `HTTP` or `HTTPS`. The default is *HTTPS*.
+|Yes
+
+|Endpoint Name
+|Automatically-generated name that identifiers the endpoint, which you can modify manually later on
+|Yes
+
+|Description
+|User-supplied description of the endpoint's purpose and use
```

Comment:
```suggestion
|User-supplied description of the endpoint's purpose and use.
```
I know it is tedious, but consider checking each set of Description column entries to be sure they use the same final punctuation, vs. none, per table